### PR TITLE
add accessor methods for properties used by will_paginate gem

### PR DESCRIPTION
**Pull request by DanielNill**

### CODE REVIEW

File: lib/elasticity/search.rb [hunk 1]
```diff
@@ -118,6 +118,26 @@ def search_results
         end
       end
 
+      # for pagination
+      def per_page
+        @search_definition.body[:size] || 10
```
Comment (divoxx) [inline lib/elasticity/search.rb]: This shouldn't default to a hardcoded 10, make it `response["hits"]["hits"].length`

File: lib/elasticity/search.rb [hunk 2]
```diff
@@ -118,6 +118,26 @@ def search_results
...
+      end
+
+      # for pagination
+      def total_pages
+        if total < per_page
+          1
+        else
+          total / per_page
+        end
```
Comment (divoxx) [inline lib/elasticity/search.rb]: You can simply do `(total / per_page.to_f).ceil`

**Comment by divoxx** [comment]
:+1:

File: lib/elasticity/search.rb [hunk 3]
```diff
@@ -118,6 +120,22 @@ def search_results
         end
       end
 
+      # for pagination
```
Comment (brupm) [inline lib/elasticity/search.rb]: these comment :faceplam:

**Pull request merged**